src/redux/compare/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './listeners';
```

Comment:
The huge compare.ts file had to be split up. I decided to put it under `redux/compare`. I'm not certain if this is the best approach since it's a first more isolated feature-slice.